File: Exemplos/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BRIDGE/consultar_ultima_transacao.dart
import 'package:intent_digital_hub/IntentDigitalHubService/BRIDGE/bridge_command.dart';

class ConsultarUltimaTransacao extends BridgeCommand {
  final String _pdv;

  ConsultarUltimaTransacao(this._pdv) : super('ConsultarUltimaTransacao');

  @override
  functionParameters() => '"pdv"' ':' '"$_pdv"';
}

File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BRIDGE/ConsultarUltimaTransacao.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE

import com.elgin.intent_digital_hub.IntentDigitalHubService.BRIDGE.BridgeCommand

class ConsultarUltimaTransacao(private val pdv: String) :
    BridgeCommand("ConsultarUltimaTransacao") {
    override fun functionParameters(): String {
        return "\"pdv\"" + ":" + "\"" + pdv + "\""
    }
}